training/train.py

```diff
@@ -120,7 +119,7 @@ def generate_manifest_id_dict(manifest_list):
     return manifest_id_dict
 
 
-def run_recommender(train_df, latent_factor):
+def run_recommender(train_df, latent_factor):  # pragma: no cover
```

Comment:
Minor nitpick, if this is multiple factors make this `latent_factors` as people who aren't familiar with the code get confused.